epoch/api/naming_system_api_usage.md

```diff
@@ -0,0 +1,95 @@
+[back](./README.md)
+# Naming System - intended usage
+
+## Introduction
+
+The way to interact with Naming System is to use HTTP API.
+To read about possible states and life cycle of a name please see [the doc](/AENS.md).
+
+The following assumes that the node exposes at address 127.0.0.1 the following ports:
+* HTTP external API: 3013
+* HTTP internal API: 3113
+
+## Name management flow
+
+Check the public key of the node first (used as account key in later API calls):
+```
+curl http://localhost:3113/v2/account/pub-key
+{"pub_key":"ak$3scLu3oJbhsdCJkDjfJ6BUPJ4M9ZZJe57CQ56deSbEXhaTSfG3Wf3i2GYZV6APX7RDDVk4Weewb7oLePte3H3QdBw4rMZw"}
+```
+
+Verify that the name that your going to claim is not taken.
+For the purpose of this example let's consider `fooo.baar.aet` name:
```

Comment:
@sennui @zp-sd  According to https://github.com/aeternity/protocol/blob/07b582eee188a15f0909d0a44ec425e5173723b7/AENS.md#registrars
names with 2 dots in between will not be allowed after https://www.pivotaltracker.com/story/show/154313327 - right?

Reply:
correct

Reply:
Tracked as https://www.pivotaltracker.com/story/show/154872121 - for later